tests/device/test_DimmerDev.py

```diff
@@ -118,3 +118,27 @@ def level_bytes(level):
                     mocked.assert_called_once_with(test_device, *expected)
                 else:
                     mocked.assert_not_called()
+
+    def test_get_flags(self, test_device):
+        # This should hijack get flags and should insert a call to
+        # EXTENDED_SET_GET
+        with mock.patch.object(IM.CommandSeq, 'add'):
+            test_device.get_flags()
+            calls = [
+                # can't figure out how to define the call to super().get_flags
```

Comment:
Mark this as a TODO so it stands out for future improvements?

(If you agree with this change, then the same should be made to tests/device/test_KeypadLincDev.py.)

Reply:
that's fine